src/components/TheUserMenu.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+          <v-menu  transition="slide-x-transition">
+          <template #activator="{ props }">
+            <v-btn
+              data-cy="user-menu"
+              class="hidden-xs-only d-flex text-body-1"
+              v-bind="props"
+              variant="text"
+            >
+              <v-icon icon="mdi-account-circle" />
```

Comment:
Aria hidden maybe on this too, or descriptive text.

Reply:
I hope to write up a style with directions for icons and to create issues to update all of our existing icons with aria attributes, but yes we want a label, aria-hidden="false" and role="img"
```suggestion
              <v-icon
                icon="mdi-account-circle"
                aria-label="Open user menu"
                aria-hidden="false"
              />
```